### PR TITLE
Es upgrade dotnet3.1

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: src/OrganisationRegistry.ElasticSearch.Projections/Body/BodyHandler.cs [hunk 1]
```diff
@@ -98,11 +97,9 @@ private void PrepareIndex(IElasticClient client, bool deleteIndex)
 
             if (!client.DoesIndexExist(indexName))
             {
-                var indexResult = client.CreateIndex(
+                var indexResult = client.Indices.Create(
                     indexName,
-                    index => index
-                        .Mappings(mappings => mappings
-                            .Map<BodyDocument>(typeName, BodyDocument.Mapping)));
+                    index => index.Map<BodyDocument>(BodyDocument.Mapping));
```
Comment (CumpsD) [inline src/OrganisationRegistry.ElasticSearch.Projections/Body/BodyHandler.cs:102]: Just wondering, the `typeName` that is not here anymore, does that have any impact?

File: src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs [hunk 2]
```diff
@@ -161,11 +161,9 @@ private void PrepareIndex(IElasticClient client, bool deleteIndex)
 
             if (!client.DoesIndexExist(indexName))
             {
-                var indexResult = client.CreateIndex(
+                var indexResult = client.Indices.Create(
                     indexName,
-                    index => index
-                        .Mappings(mappings => mappings
-                            .Map<OrganisationDocument>(typeName, OrganisationDocument.Mapping)));
+                    index => index.Map<OrganisationDocument>(OrganisationDocument.Mapping));
```
Comment (CumpsD) [inline src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs:166]: Same question:
> Just wondering, the `typeName` that is not here anymore, does that have any impact?

File: src/OrganisationRegistry.ElasticSearch.Projections/People/Handlers/Person.cs [hunk 3]
```diff
@@ -107,11 +107,9 @@ private void PrepareIndex(IElasticClient client, bool deleteIndex)
 
             if (!client.DoesIndexExist(indexName))
             {
-                var indexResult = client.CreateIndex(
+                var indexResult = client.Indices.Create(
                     indexName,
-                    index => index
-                        .Mappings(mappings => mappings
-                            .Map<PersonDocument>(typeName, PersonDocument.Mapping)));
+                    index => index.Map<PersonDocument>(PersonDocument.Mapping));
```
Comment (CumpsD) [inline src/OrganisationRegistry.ElasticSearch.Projections/People/Handlers/Person.cs:112]: Same question:
> Just wondering, the `typeName` that is not here anymore, does that have any impact?

File: src/OrganisationRegistry.ElasticSearch/People/PersonDocument.cs [hunk 4]
```diff
@@ -24,8 +25,7 @@ public class PersonDocument : IDocument
                     .Name(p => p.ChangeId))
 
                 .Date(d => d
-                    .Name(p => p.ChangeTime)
-                    .Format("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSZZ||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis"))
+                    .Name(p => p.ChangeTime))
```
Comment (CumpsD) [inline src/OrganisationRegistry.ElasticSearch/People/PersonDocument.cs:28]: What's the error that caused you to remove this `.Format()`?

File: src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs [hunk 5]
```diff
@@ -29,8 +30,7 @@ public class OrganisationDocument : IDocument
                     .Name(p => p.ChangeId))
 
                 .Date(d => d
-                    .Name(p => p.ChangeTime)
-                    .Format("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSZZ||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis"))
+                    .Name(p => p.ChangeTime))
```
Comment (CumpsD) [inline src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs:33]: What's the error that caused you to remove this `.Format()`?

File: src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs (hunk 2 above)
Comment (koenmetsu) [reply src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs:166]: Types are gone from ES 7.x

**Pull request merged**